tex/chapters/literature.tex

```diff
@@ -1,11 +1,106 @@
+
+The notation $JA$ comes from the initials of the names Joss and Anne.
+Joss was a strategy submitted to one of Axelrod’s original tournaments and it would occasionally defect without provocation in the hopes of a slight improvement in score.
+Anne is the first name of A. Stanley who suggested the addition of random cooperation (refs from ashlock paper) instead of random defection.
+When $x + y = 1$, the original strategy is not used, and the resulting behavior is a random strategy with probabilities $(x, y)$.
+In more general terms, a $JA$ strategy is an alteration of a strategy $A$ that causes the strategy to be played with random noise inserted into the responses.
+
+\begin{definition}\label{def:fingerprint}
+A \textbf{Fingerprint} $F_A(S, x, y)$ with $0 \leq x, y \leq 1$, $x+y \leq 1$ for strategy $S$ and probe $A$, is the function that returns the expected score of strategy $S$ against $JA(A, x, y)$ for each possible $(x, y)$.
+\end{definition}
+
+\begin{definition}\label{def:double-fingerprint}
+The \textbf{Double Fingerprint} $F_{AB}(S, x, y)$ with $0 \leq x, y \leq 1$ returns the expected score of strategy $S$ against $JA(A, x, y)$ if $x+y \leq 1$, and $JA(B, 1-y, 1-x)$ if $x+y \geq 1$.
+\end{definition}
+
+\begin{definition}\label{def:dual}
+Strategy $A'$ is said to be the \textbf{Dual} of strategy $A$ if $A$ and $A'$ can be written as finite-state machines that are identical except that their responses are reversed.
+\end{definition}
+
+\begin{theorem}\label{fingerprint-unit-square}
+If $A$ and $A'$ are dual strategies, then $F_{AA'}(S, x, y)$ is identical to the function $F_A(S, x, y)$ extened over the unit square.
+\end{theorem}
+
+
+\section{Example Fingerprint Construction}
+
+There are several steps to constructing the Fingerprint of a strategy a basic familiarity of Markov Chains is required.
+An outline of the steps is as follows:
+
+\begin{itemize}
+    \item Build the markov chain for IPD between the strategy and probe strategy.
+    \item Construct the corresponding transition matrix.
+    \item Find the steady state distribution.
+    \item Calculate the overall expected score by taking the dot product of the steady state distribution with the payoff vector.
+    \item Plot the resulting function. 
+\end{itemize}
+
+We will now apply this process in order to obtain a fingerprint for the strategy Win-Stay-Lose-Shift (sometimes referred to as Pavlov) when probed by Tit-For-Tat.
+
+\textbf{Step 1} - Build the markov chain.
+
+\textbf{Step 2} - Construct the transition matrix.
+
+\begin{equation}\label{eq:transition_matrix}
+% 
+T = \bordermatrix{~      & (C, C) & (C, D) & (D, C) & (D, D) \cr
+                  (C, C) & 1 - y  & 0      & 0      & x      \cr
+                  (C, D) & y      & 0      & 0      & 1 - x  \cr
+                  (D, C) & 0      & 1 - y  & x      & 0      \cr
+                  (D, D) & 0      & y      & 1 - x  & 0}
+% 
+\end{equation}
+
+\textbf{Step 3} - Find the steady state distribution.
```

Comment:
Need a sentence or two here explaining how this was obtained (or write out some of the algebra).